app/_de_tools/server.md

```diff
@@ -1,18 +1,23 @@
 ---
 number: 11
-header: "Have your server at home"
-time: "10h"
-skills: 5
-spy_reduction: 2
-incomfort: 4
+header: "Einen eigenen Server zu Hause haben"
+time: "8h"
+skills: 3
```

Comment:
I still think it is a 5.
Just read this page:
https://ubos.net/

And imagine my dad (I don't know yours) reading that.
He dropped at the first line: `Linux distro`

Reply:
Yes, but 5 years I have linux, I know Bash, PHP, CSS :-) 
Depends on motivation I think.

Many Lambda users went to install Yunohost on their RPI and most of them achieve their goal (having a nextcloud for example). I can change that to 4 but 5 is a bit too much. :-) Try Yunohost or FreedomBox and you will see. ;)

Reply:
Yes, I know these projects, I agree.
Let's cut it in `4` then :)
And hopefully in the future, we can further decrease.